File: app/src/main/java/com/pluu/webtoon/api/Detail.java
package com.pluu.webtoon.api;

import java.util.List;

/**
 * 에피소드 상세 내용 Item Class
 * Created by nohhs on 2015-03-12.
 */
public class Detail {
	public String title;
	public String webtoonId, episodeId;
	public String nextLink, prevLink;
	public int nextIdx, prevIdx;
	public List<DetailView> list;

	// ERROR_TYPE
	public ERROR_TYPE errorType;
	public String errorMsg;
}

File: domain/src/main/java/com/pluu/webtoon/domain/moel/Detail.kt
package com.pluu.webtoon.domain.moel

sealed class DetailResult {

    /**
     * 에피소드 상세 내용 Item Class
     * Created by pluu on 2018. 10. 07..
     */
    class Detail(
        var webtoonId: String,
        var episodeId: String
    ) : DetailResult() {
        var title: String? = null
        var nextLink: String? = null
        var prevLink: String? = null
        var list: List<DetailView> = emptyList()
    }

    /**
     * 에피소드 에러 Class
     * Created by pluu on 2018. 10. 07..
     */
    class ErrorResult(
        val errorType: ERROR_TYPE
    ) : DetailResult()
}
